backend/api/api_deployment_views.py

```diff
@@ -74,28 +74,29 @@ def get(
         self, request: Request, org_name: str, api_name: str, api: APIDeployment
     ) -> Response:
         execution_id = request.query_params.get("execution_id")
-        include_metadata = request.query_params.get("include_metadata", False)
+        include_metadata = (
```

Comment:
NIT: Try to use [this util function](https://github.com/Zipstack/unstract/blob/c694de72e7a2cb051ed008956287df8a77587233/backend/utils/common_utils.py#L6-L15)

Reply:
Noted. Will use this util and raise a separate PR when I get time.